async/README.md

```diff
@@ -1,6 +1,6 @@
 # debounce
 
-Makes the function run after the given period of not being called. Useful to delay input submission for autocomplete etc.
+Makes the function run after the given period of not being called. Useful to delay input submission for Makes the function run after the given period of not being called. Useful to delay input submission for auto-complete etc.
```

Comment:
what just happened here :)?

Reply:
Thanks, this was an editing error. Fixed :)